### PR TITLE
Update _config.yml

### PR DESCRIPTION
**Pull request by jekyllbot**
PR automatically created for @pathawks.

```text
Update _config.yml

Redcarpet is not used much for Markdown anymore, but it would be really
helpful to be able to test things that require `site.url` and
`site.baseurl` to be set.
```

### CODE REVIEW

File: generate_site.sh [hunk 1]
```diff
@@ -23,7 +23,8 @@ cp ./default_layout.html $LAYOUT_DIR/default.html
 cp ./post_layout.html $LAYOUT_DIR/post.html
 
 cat >${SITE_DIR}/_config.yml << EOF
-markdown: redcarpet
+url: https://example.com
+baseurl: baseURL
```
Comment (DirtyF) [inline generate_site.sh]: should be `/baseURL` 😄

**Review by DirtyF — COMMENTED** [review]
👍 minus the missing slash on `baseurl`

File: generate_site.sh (hunk 1 above)
Comment (pathawks) [reply generate_site.sh]: Doesn’t matter when using our URL filters, but you are right :+1:

**Review by ashmaroli — APPROVED** [review]

**Comment by pathawks** [comment]
Fixed :+1:

**Pull request merged**